src/main/java/com/charu/Operations.java

```diff
@@ -0,0 +1,77 @@
+package com.charu;
+import java.util.Scanner;
+
+import java.util.*;
+public class Operations {
+    Tasks task ;
+    Scanner myObj = new Scanner(System.in);
+    ArrayList<Tasks> lists = new ArrayList<Tasks>();
+    int counter;
+    public void add(){
+     if(!lists.isEmpty()){
+    counter+=lists.get(lists.size()-1).getId();
```

Comment:
This should be `newTaskId` and not `counter`

Reply:
addressed